### DISCUSSION TITLE
Request: show icon on dashboard indicating that data has been excluded from a chart due to row limit

### DISCUSSION THREAD

**Discussion by sfirke** [apache/superset#26681] [Ideas]
Sometimes a chart grows bigger than the initial row limit I set (say 10k records) and it begins showing a version that has some data silently missing.  This [PR from April 2024](https://github.com/apache/superset/pull/27700) does a nice job of this in chart view, with a red icon that has an informative tooltip.

However, most of the time my users view a chart on the dashboard, not directly in chart editor view.  I would like an icon with informative tooltip to be present on the dashboard, too.

Here is a chart that is in that silently failed state:
![image](https://github.com/apache/superset/assets/7569808/77fc859d-e983-42b0-be04-28f9020a4ded)

Here's the same chart when I increased the row limit from 10k to 100k:
![image](https://github.com/apache/superset/assets/7569808/41f4a626-20ce-4dca-97f7-844928fd258a)

*Edited to reflect the state of things following #27700*

**Comment by michael-s-molina** [comment]
+1
